mlir/lib/CAPI/IR/BuiltinAttributes.cpp

```diff
@@ -10,6 +10,7 @@
 #include "mlir-c/Support.h"
 #include "mlir/CAPI/AffineMap.h"
 #include "mlir/CAPI/IR.h"
+#include "mlir/CAPI/IntegerSet.h"
```

Comment:
this is already include in BuiltinAttributes.h why including it also here?

Reply:
Thank you for the review!   The reason this is needed is because CAPI/IntegerSet.h includes the following:

```c++
 18 #include "mlir-c/IntegerSet.h"                                                                                          
 19 #include "mlir/CAPI/Wrap.h"                                                                                             
 20 #include "mlir/IR/IntegerSet.h"         
```

Including it for the sake of picking up mlir-c/IntegerSet.h is not needed as you pointed out since this is included by BuiltinAttributes.h.  However, we also need the mlir/IR/IntegerSet.h.  Following the AffineMap.h convention of earlier work, I include CAPI/IntegerSet.h.